CONTRIBUTING.md

```
@@ -107,41 +107,37 @@ Visual regression tests are run in a docker image (to be consistent on CI enviro
 NB: To expose host network on MacOS and Windows you [need](https://docs.docker.com/engine/network/tutorials/host/#prerequisites) to enable it manually in Docker settings.
 
 ```sh
-docker run --rm -v ${PWD}:/work/ -w /work/ -it --network host --entrypoint /bin/bash "mcr.microsoft.com/playwright:v$(grep -m1 '^  playwright:' .yarnrc.yml | awk '{print $2}' | tr -d '^')"
+docker run --rm -v ${PWD}:/work/ -w /work/ -it --network host --entrypoint /bin/bash "mcr.microsoft.com/playwright:v$(grep -m1 '"playwright":' package.json | sed -E 's/.*"([0-9.]+)".*/\1/')"
 ```
 
-2. (in Docker) Install platform-specific binaries (until `sharp` [utilize](https://github.com/lovell/sharp/issues/3750) `supportedArchitectures` in `.yarnrc.yml`)
+2. (in Docker) Install Bun and reinstall platform-specific binaries (the image's `node_modules` was built for the host platform, not Linux)
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)